packages/origin-backend-client-mocks/src/OffChainDataSourceMock.ts

```diff
@@ -8,36 +8,44 @@ import {
     RequestClient,
     IOrganizationClient,
     IDemandClient,
-    IEventClient
-} from "@energyweb/origin-backend-client";
+    IEventClient,
+    IFilesClient
+} from '@energyweb/origin-backend-client';
 
-import { PreciseProofClientMock } from "./PreciseProofClientMock";
-import { ConfigurationClientMock } from "./ConfigurationClientMock";
-import { UserClientMock } from "./UserClientMock";
-import { DeviceClientMock } from "./DeviceClientMock";
-import { OrganizationClientMock } from "./OrganizationClientMock";
-import { DemandClientMock } from "./DemandClientMock";
-import { EventClientMock } from "./EventClientMock";
+import { PreciseProofClientMock } from './PreciseProofClientMock';
+import { ConfigurationClientMock } from './ConfigurationClientMock';
+import { UserClientMock } from './UserClientMock';
+import { DeviceClientMock } from './DeviceClientMock';
+import { OrganizationClientMock } from './OrganizationClientMock';
+import { DemandClientMock } from './DemandClientMock';
+import { EventClientMock } from './EventClientMock';
 
 export class OffChainDataSourceMock implements IOffChainDataSource {
     dataApiUrl: string;
 
     preciseProofClient: IPreciseProofClient = new PreciseProofClientMock();
+
     configurationClient: IConfigurationClient = new ConfigurationClientMock();
+
     userClient: IUserClient = new UserClientMock();
+
     eventClient: IEventClient = new EventClientMock();
-    
+
     demandClient: IDemandClient;
+
```

Comment:
Am I the only one who finds this rule annoying? 😅

Reply:
I'm neutral towards it. ; p I can format it with one keyword so doesn't make much difference for me. ;p